vllm/envs.py

```diff
@@ -213,6 +214,11 @@
     # Both spawn and fork work
     "VLLM_WORKER_MULTIPROC_METHOD":
     lambda: os.getenv("VLLM_WORKER_MULTIPROC_METHOD", "spawn"),
+
+    # Timeout for fetching images when serving multimodal models
+    # Default is 2 seconds
+    "VLLM_IMAGE_FETCH_TIMEOUT":
+    lambda: int(os.getenv("VLLM_IMAGE_FETCH_TIMEOUT", "2")),
```

Comment:
A quick look at [the requests documentation](https://requests.readthedocs.io/en/latest/user/advanced/#timeouts) suggests that we should set the timeout to at least 3 seconds.